pcp/Platform.c

```diff
@@ -350,7 +350,7 @@ bool Platform_init(void) {
    if (opts.context == PM_CONTEXT_ARCHIVE) {
       gettimeofday(&pcp->offset, NULL);
 #if PMAPI_VERSION >= 3
-      struct timeval start = { opts.start.tv_sec, opts.start.tv_nsec / 1000 };
+      struct timeval start = { opts.start.tv_sec, (int) (opts.start.tv_nsec / 1000) };
```

Comment:
Don't expect the type `suseconds_t` to be always synonym of `int`
```suggestion
      struct timeval start = { opts.start.tv_sec, (suseconds_t)(opts.start.tv_nsec / 1000) };
```